src/app/Models/Traits/HasRelationshipFields.php

```diff
@@ -71,7 +75,7 @@ public static function isColumnNullable($column_name)
         $table = $instance->getTableWithPrefix();
 
         // MongoDB columns are alway nullable
-        if ($conn->getConfig()['driver'] === 'mongodb') {
+        if (!in_array($conn->getConfig()['driver'], CRUD::getSqlDriverList()) {
```

Comment:
```suggestion
        if (!in_array($conn->getConfig()['driver'], CRUD::getSqlDriverList())) {
```

Missing parentheses.

Reply:
Cherry picked the wrong commit. I had this fixed :| 

Sorry for that.